documentation/periodic_builds.md

```diff
@@ -0,0 +1,65 @@
+Periodic Builds
+--------------------------
+
+Ocim includes a tool to automatically spawn and provision new AppServers at
+time intervals. The main use case for this is an adhoc continuous integration
+service. The Open edX platform spans many dependencies, so rebuilding on new
+commits to a repository isn't enough to catch possibly breaking changes
+elsewhere in the dependency chain.
+
+## Configuration
+
+The behaviour of periodic builds can be configured per-openedxinstance from the Django
```

Comment:
Nit: `per-openedxinstance` can be changed to `per openedx instance` for better readability.